src/Tizen.NUI.Components/Controls/FlexibleView/FlexibleView.Helper.cs

```diff
@@ -181,6 +181,32 @@ public FlexibleViewViewHolder GetChildAt(int index)
                 return mViewList[index];
             }
 
+            protected override void Dispose(DisposeTypes type)
+            {
+                if (disposed)
+                {
+                    return;
+                }
+
+                if (type == DisposeTypes.Explicit)
+                {
+                    Clear();
+
+                    if (mTapGestureDetector != null)
+                    {
+                        mTapGestureDetector.Detected -= OnTapGestureDetected;
+                        mTapGestureDetector.Dispose();
+                        mTapGestureDetector = null;
+                    }
+
+                    if (mFlexibleView != null)
```

Comment:
In my opinion,
A FlexibleView owns a ChildHelper instance and ChildHelper.mFlexibleView is just a pointer to its owner. So that the dispose ownership of mFlexibleView is not held by ChildHelper.

[CA1001](https://docs.microsoft.com/en-us/visualstudio/code-quality/ca1001?view=vs-2019)
> It is OK to suppress the warning when the dispose ownership of the field(s) is not held by the containing type.

I think instead of disposing mFlexibleView here, it needs to dispose FlexibleView.mChildHelper in FlexibleView.Dispose().

        protected override void Dispose(DisposeTypes type)
        {
            ......
                if (mChildHelper != null)
                {
                    mChildHelper.Clear();
                    mChildHelper.Dispose();
                    mChildHelper = null;
                }
            }
            base.Dispose(type);
        }


What do you think?

Reply:
I think you are right! I will update the code.